### PR TITLE
Rc bikbal ross

### PR DESCRIPTION
**Pull request by soldnermike**

### CODE REVIEW

**Comment by soldnermike** [comment]
#review

File: livefyre-apps/assets/js/lfapps.js [hunk 1]
```diff
@@ -1,32 +1,28 @@
-var LFAPPS = {};
+var Livefyre = Livefyre || {};
```
Comment (ross-pfahler) [inline livefyre-apps/assets/js/lfapps.js]: ``Livefyre = window.Livefyre || {};``

Comment (gobengo) [inline livefyre-apps/assets/js/lfapps.js]: Better yet, use your own global if you really need one

Comment (ross-pfahler) [inline livefyre-apps/assets/js/lfapps.js]: @gobengo I told him not to :)

Comment (ross-pfahler) [inline livefyre-apps/assets/js/lfapps.js]: we should discuss.

**Pull request merged**